packages/@ember/-internals/glimmer/tests/integration/custom-component-manager-test.js

```diff
@@ -524,6 +525,11 @@ moduleFor(
     }
 
     '@test capabilities helper function must be used to generate capabilities'(assert) {
+      if (!DEBUG) {
+        assert.expect(0);
+        return;
+      }
```

Comment:
Do we have a "pragma" for this (e.g. `@prod-test` or some such)?

Reply:
I don't think so, but we should probably add one